spec/operators/timestamp-spec.ts

```diff
@@ -0,0 +1,147 @@
+import * as Rx from '../../dist/cjs/Rx.KitchenSink';
+declare const {hot, cold, asDiagram, expectObservable, expectSubscriptions};
+
+declare const rxTestScheduler: Rx.TestScheduler;
+const Observable = Rx.Observable;
+
+/** @test {timestamp} */
+describe('Observable.prototype.timestamp', () => {
+  asDiagram('timestamp')('should record the time stamp per each source elements', () => {
+    const e1 = hot('--a--^b-c-----d--e--|');

```

Comment:
I think the PNG marble diagram generation won't like the `--a--^b`, so could you just start this hot observable from the beginning? I'm also not sure if PNG diagrams need `e1subs`, so maybe that can be removed as well.